ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcAcidRowBatchReader.java

```diff
@@ -129,6 +137,16 @@
    */
   private SearchArgument deleteEventSarg = null;
 
+  /**
+   * Cachetag associated with the Split
+   */
+  private final CacheTag cacheTag;
+
+  /**
+   * Skip using Llap IO cache for checking delete_delta files if the configuration is not correct
+   */
+  private static boolean skipLlapCache = true;
```

Comment:
Initialized to true on purpose for now? If not, I don't see it getting set to false.

Reply:
That was a mistake. Corrected, and initialized as false